thunder/executors/torch_compile.py

```diff
@@ -84,7 +85,12 @@ def torch_interpreted_func(*args):
     # torch.compile executor"
     torch_trace = trace(inline_trace=False)(torch_interpreted_func, *sorted_unique_inputs)
     trace_callable = torch_trace.python_callable(include_decorators=False)
-    compiled_func = torch.compile(trace_callable, fullgraph=True)
+    torch_compile_fullgraph: None | bool = get_compile_option(
```

Comment:
Adding a compile option for this is a cool debugging idea.

Whether torch.compile generates a single graph or not for what we export to it may be considered an implementation detail, but this setting could also be useful to set for debugging.